doc/users/credits.rst

```diff
@@ -17,157 +19,290 @@ git revision control history of the project:
 Ben Root,
+Benedikt Daurer,
+Benedikt J. Daurer,
+Benjamin Berg,
+Benjamin Congdon,
 Benjamin Reedlunn,
+Bernhard M. Wiedemann,
+Bianca Gibson,
+Big Data Tech. Lab 이태훈,
 Binglin Chang,
+Björn Dahlgren,
+Blaise Thompson,
+Boaz Mohar,
 Bradley M. Froehle,
 Brandon Liu,
+Brendan Zhang,
+Brennan Magee,
 Brett Cannon,
 Brett Graham,
 Brian Mattern,
 Brian McLaughlin,
 Bruno Beltran,
+Bruno Zohreh,
 CJ Carey,
 Cameron Bates,
 Cameron Davidson-Pilon,
+Cameron Fackler,
 Carissa Brittain,
 Carl Michal,
 Carwyn Pelley,
 Casey Webster,
 Casper van der Wel,
 Charles Moad,
+Charles Ruan,
+Chen Karako,
 Chris Beaumont,
 Chris G,
+Chris Holdgraf,
 Christian Brueffer,
 Christian Stade-Schuldt,
 Christoph Dann,
+Christoph Deil,
 Christoph Gohlke,
 Christoph Hoffmann,
+Christopher Holdgraf,
 Cimarron Mittelsteadt,
+Cody Scot,
+Colin,
+Conner R. Phillips,
 Corey Farwell,
+Craig Citro,
 Craig M,
 Craig Tenney,
+DaCoEx,
+Dakota Blair,
+Damian,
 Damon McDougall,
 Dan Hickstein,
+Dana,
+Daniel C. Marcu,
 Daniel Hyams,
+Daniel Laidig,
 Daniel O'Connor,
+Danny Hermes,
 Dara Adib,
 Darren Dale,
+DaveL17,
+David A,
 David Anderson,
 David Haberthür,
 David Huard,
 David Kaplan,
+David Kent,
 David Kua,
-David Trémouilles,
+David Stansby,
+David Trémouilles,
 Dean Malmgren,
+Derek Kim,
+Derek Tropf,
+Devashish Deshpande,
+Diego Mora Cespedes,
+Dietmar Schwertberger,
+DietmarSchwertberger,
+Dietrich Brunn,
+Divyam Madaan,
 Dmitry Lupyan,
+Dmitry Shachnev,
 DonaldSeo,
 Dora Fraeman,
+DoriekeMG,
+Dorota Jarecka,
+Doug Blank,
+Drew J. Sonne,
 Duncan Macleod,
+Dylan Evans,
+E. G. Patrick Bos,
 Edin Salkovic,
+Egor Panfilov,
 Elena Glassman,
 Elias Pipping,
+Elijah Schutz,
 Elliott Sales de Andrade,
+Elvis Stansvik,
 Emil Mikulic,
+Emlyn Price,
 Eric Dill,
 Eric Firing,
+Eric Larson,
 Eric Ma,
 Eric O. LEBIGOT (EOL),
+Eric Wieser,
 Erik Bray,
+Erik M. Bray,
+Erin Pintozzi,
 Eugen Beck,
 Eugene Yurtsev,
 Evan Davey,
 Ezra Peisach,
+Fabian Kloosterman,
+Fabian-Robert Stöter,
 Fabien Maussion,
 Fabio Zanini,
+FedeMiorelli,
 Federico Ariza,
 Felipe,
+Felix,
+Felix Kohlgrüber,
+Felix Yan,
 Fernando Perez,
+Filip Dimitrovski,
 Filipe Fernandes,
+Florencia Noriega,
+Florian Le Bourdais,
 Florian Rhiem,
 Francesco Montesano,
 Francis Colas,
+Franco Vaccari,
+Francoise Provencher,
+Frank Yu,
 François Magimel,
+Gabe,
+Gabriel Munteanu,
+Gauravjeet,
 Gaute Hope,
 Gellule Xg,
+Geoffrey Spear,
 Geoffroy Billotey,
 Gerald Storer,
 Giovanni,
 Graham Poulter,
 Gregory Ashton,
 Gregory R. Lee,
+Grillard,
 Grégory Lielens,
 Guillaume Gay,
+Guillermo Breto,
 Gustavo Braganca,
+Gustavo Goretkin,
+HHest,
+Hajoon Choi,
+Hakan Kucukdereli,
 Hans Dembinski,
 Hans Meine,
 Hans Moritz Günther,
+Harshit Patni,
 Hassan Kibirige,
+Hastings Greer,
+Heath Henley,
+Heiko Oberdiek,
+Helder,
+Henning Pohl,
+Herbert Kruitbosch,
 Holger Peters,
 Hubert Holin,
 Ian Thomas,
+Ida Hjorth,
 Ignas Anikevicius (gns_ank),
 Ilia Kurenkov,
+Ilya Flyamer,
+ImSoErgodic,
+Importance of Being Ernest,
+ImportanceOfBeingErnest,
```

Comment:
@ImportanceOfBeingErnest which one of these do you prefer? (this just needs to be added as an entry in the .mailmap file)

Reply:
I guess the one without spaces is better, it makes it more obvious that this is a username, and not a user name.